app/src/pages/inside/manualLaunchesPage/manualLaunchExecutionPage/executionCommentSection/executionCommentSection.scss

```
@@ -147,44 +176,49 @@
+      max-width: 400px !important;
+      margin-top: 0 !important;
+      grid-template-columns: none !important;
+    }
   }
+}
 
-  &__remove {
-    flex-shrink: 0;
-    align-self: center;
+.execution-comment-section__attachments {
+  :global([class*='file-drop-area--overlay']) :global([class*='file-drop-area']) {
```

Stylelint (17.11.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)